src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing Cloud Credentials, a central location to manage your authentication with AWS and GCP  <!-- 8550 -->
+- Enabling Okteto Insights for all users of Okteto <!-- 8627 -->
+- Added configurable buildkit network mode  <!-- 8486 -->
+- Init dynamic config  <!-- https://github.com/okteto/app/pull/8475 -->
+- Added CLI, Chart, and Okteto App version to our API headers  <!-- 8545 -->
+- Support to enable/disable resource quotas for individual Namespaces  <!-- 8604 -->
```

Comment:
This change is related to allow to enable only specific resource quotas instead of all of them for namespaces. I mean, before, if you enabled quotas, you would restrict all the resources of the quotas (pods, services, secrets, configmaps, etc..)

Now, with this change, you can enable the quotas and restrict only one or a subset of resources, allowing to create any amount of the other resources. For example, you could restrict only the number of pods.

I think it could be relevant to highlight here that we have changed the default values. If there was a user that enabled only the quotas, they were using the previous default values that now are `-1`. 

We could also link to here: https://www.okteto.com/docs/1.25/self-hosted/helm-configuration/#quotas